Edit README (#2)
nikomatsakis/a-mir-formality: README.md

# a-mir-formality

> This repository aims to be a complete, authoritative formal model of the [Rust MIR](https://rustc-dev-guide.rust-lang.org/mir/index.html).
> At present, however, it is an incomplete, experimental model thereof.

Author: Could combine these sentences maybe:

```suggestion
This repository is an early-stage experimental project that aims to be a complete, authoritative formal model of the [Rust MIR](https://rustc-dev-guide.rust-lang.org/mir/index.html).
```
